content/spin/v2/http-outbound.md

```diff
@@ -224,3 +225,7 @@ You must still grant permission by including `self` in `allowed_outbound_hosts`:
 ```toml
 allowed_outbound_hosts = ["http://self", "https://self"]
 ```
+
+### Private Endpoints
+
+Private endpoints represent internal microservices that are shielded from external network exposure. These endpoints do not have public HTTP routes and can only be accessed internally within the application's architecture.
```

Comment:
It should be related to the service chaining feature. So an example could be that two components use this endpoint internally, through service chaining. Maybe it's not it's oiwn heading, but just a call-out in the service chaining section, that these components have endpoints, which are not exposed?

Reply:
Thanks @mikkelhegn I have updated this page and this will be updated in the next push.